_script/cibuild

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env bash
 
-set -exuo pipefail
 cd $(dirname $0)/..
+source _script/requirements
+set -exuo pipefail
```

Comment:
why you change the order of the "set"?

Reply:
I'm interested the "injected script" to have its own control of how is configuring bash, but once the previous line returns, I want to have control of the configuration applying for the rest of this bash script

Reply:
ack